src/main/scala/epic/logo/ArgmaxInferencer.scala

```diff
@@ -0,0 +1,7 @@
+package epic.logo
+
+trait ArgmaxInferencer[T, Y, W] {
+
+  def argmax(weights : Weights[W], instance : T) : (Y, FeatureVector[W], Double)
```

Comment:
hey malcolm, you're missing some context. we need this internally soon and i doubt it will last all that long in this state.